templates/search/blocks/search_results_tag_frequency.html

```diff
@@ -35,7 +35,7 @@
                 <a href="{% tag_type_url tag_type=tag_type.ORGANISATION|upper %}#tagTop" class="ohos-tag ohos-tag--organisation">
 
                     <span class="ohos-tag__inner">
-                        Organisation
+                        <label for="Category: Organisation">Organisation</label>
```

Comment:
Label is usually associated with an input field. (same browser error as label for Category Location

Reply:
now using aria-label on <a href> tags